runtime/compiler/control/JITaaSCompilationThread.hpp

```diff
@@ -28,6 +28,10 @@
 #include "rpc/J9Client.hpp"
 #include "env/PersistentCollections.hpp"
 
+#define MAJOR_NUMBER 0
```

Comment:
It's better if we attach some size to these for instance `static const uint_8 MAJOR_NUMBER = 0`